config/locales/views/admin/en.yml

```diff
@@ -6,3 +6,5 @@ en:
     hidden: Hidden
     visible: Visible
   markus_administration: MarkUs Administration
+  users:
+    admin_csv_upload_help_html: For more information about this feature and the<br>file format(s) supported, please visit <a href="https://github.com/MarkUsProject/Wiki/blob/%{markus_version}/Admin-Guide.md#bulk-uploading-of-end-users">this page</a>.
```

Comment:
Not sure if this should link to the section within the Admin Guide describing this feature and its expected format or if I should create a new `Admin-Guide--Importing-and-Exporting-Data.md` that describes the expected csv file format and link to that.